src/library/lastplayedcache.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <QObject>
+#include <QSet>
+#include <QSqlDatabase>
+#include <QString>
+
+#include "track/track.h"
+#include "track/trackid.h"
+
+class TrackCollection;
+
+const QString LASTPLAYEDTABLE_NAME = "last_played";
```

Comment:
This will create multiple instances of this constant for everyone who includes the header file. For a singular instance declare this constant as `extern` and define it only one in the .cpp file.

Reply:
this pattern is all over mixxx fyi, see trackschema.h, defs_vinylcontrol.h, etc.  because I have a #pragma once, I don't think it'll get redefined, right?  Anyway, fixed